features/step_definitions/logging.rb

```diff
@@ -1289,3 +1292,23 @@
   end
   cb[collector_name] = fluentd_component_label
 end
+
+Given /^I check if the remaining_resources in woker nodes meet the requirements for logging stack$/ do
```

Comment:
What's `remaining_resources` ? Did you mean `free resources` ?

Reply:
Yes, it's defined in https://github.com/openshift/verification-tests/blob/master/lib/openshift/node.rb#L252-L263

Reply:
Thanks for the confirm.
Ack